cloud/passportauth/integration/UserSeedData.ts

```diff
@@ -0,0 +1,70 @@
+export interface User {
```

Comment:
We can include here methods to map username and password in order to not hardcode them :D